src/main/java/com/okta/sdk/clients/AuthApiClient.java

```diff
@@ -114,6 +115,19 @@ public AuthResult authenticate(String username, String password, String relaySta
     }
 
     /**
+     * Authenticate into organization.
+     *
+     * @param  token {@link String}                 activationToken to use for authentication.
+     * @return {@link AuthResult}                   Result of the authentication transaction.
+     * @throws IOException                          If an input or output exception occurred.
+     */
+    public AuthResult authenticate(String token) throws IOException {
```

Comment:
Should we call this authenticateWithActivationToken, like we have authenticateWithFactor below? Also as a nit, can you change the description from "Authenticate into organization" to "Authenticate with activation token"?